brangelina.vim

```diff
@@ -72,6 +72,8 @@ let g:ale_elm_make_use_global=1
 let g:deoplete#enable_at_startup = 1
 let g:asyncrun_open = 8
 let g:neoterm_autoscroll = 1
+let g:neoterm_autoinsert = 1
+let g:neoterm_default_mod = 'tab'
```

Comment:
@jwoudenberg how do you like this?

Reply:
I still prefer full screen, failing that a split screen.

Sounds like something we could easily overwrite in our init files though, so also okay with this.